component/libfuse/libfuse2_handler.go

```diff
@@ -240,6 +240,13 @@ func (lf *Libfuse) destroyFuse() error {
 
 //export libfuse2_init
 func libfuse2_init(conn *C.fuse_conn_info_t) (res unsafe.Pointer) {
+	// fuse started processing requests, no error happened during mounting
+	log.Trace("Libfuse::libfuse2_init : notifying parent")
```

Comment:
SA Issue:
Below issues are found in SA
component/libfuse/libfuse2_handler.go:247:15: Error return value of `syscall.Kill` is not checked (errcheck)
		syscall.Kill(ppid, syscall.SIGUSR2)
		            ^
cmd/mount.go:424:27: Error return value of `daemon.ServeSignals` is not checked (errcheck)
				go daemon.ServeSignals()